src/convos/index.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: convos-intro
+---
+
+# Conversation Management
+
+Abbot helps you manage conversations with your customers in Slack.
+So, what is a conversation?
+In Abbot, a conversation is a set of chat messages that represent a single request, question or comment from a customer.
```

Comment:
```suggestion
In Abbot, a Conversation is a set of chat messages that represent a single request, question or comment from a customer.
```

Reply:
There are many instances of capitalized and uncapitalized "Conversation" in the docs. We should pick one way and stick with it (my vote is for capitalized since it describes a feature we offer).